shopify_python/google_styleguide.py

```diff
@@ -322,3 +328,14 @@ def __lambda_func(self, node):  # type: (astroid.Lambda) -> None
                     lambda_fun = "lambda " + left + ', ' + right + ": " + " ".join([left, node.ops[0][0], right])
                     op_fun = "operator." + operator
                     self.add_message('lambda-func', node=node, args={'op': op_fun, 'lambda_fun': lambda_fun})
+
+    def __class_def_check(self, node):  # type: (astroid.ClassDef) -> None
+        """Enforce a blank line after a class definition line."""
+        if isinstance(node, astroid.ClassDef):
```

Comment:
Is this check necessary? Shouldn't `visit_classdef` only be called at a `ClassDef` line?